packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/clients/UserAuthenticators.java

```diff
@@ -17,8 +17,8 @@ public UserAuthenticators(AuthNClient.Builder builder) {
 	 * Delete user authenticator
 	 * @pangea.description Delete user authenticator.
 	 * @pangea.operationId authn_post_v2_user_authenticators_delete
-	 * @param request
-	 * @return UserAuthenticatorsDeleteResponse
+	 * @param request Request parameters.
+	 * @return An empty object.
```

Comment:
Same comment that in Dotnet

Reply:
This is just the Javadoc. I didn't touch the actual return type.